src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -422,11 +409,16 @@ public String getSource() {
     }
 
     public String getAuthorEmail() {
-        return authorEmail;
+        try {
+            return pr.getUser().getEmail();
+        } catch (IOException e) {
+            logger.log(Level.SEVERE, "Unable to fetch author info for " + id);
```

Comment:
Is it a concern that the number of round trips to GH will cause rate limit failures?